2016-2017_Starstruck/Module Prototypes/include/LCD.h

```diff
@@ -19,7 +21,7 @@ typedef struct MenuItem {
  * This function should be called from initialize()
  * DO NOT initialize the LCD separately
  */
-MenuItem *lcdmInit();
+MenuItem *lcdmInit(FILE *lcdPort);

```

Comment:
Yea. If you return the root node by value instead of by reference, its location changes. So if I create a root node and set its next and previous to itself, passing it by value breaks it bc the new one is at a different location with next and previous pointing to a memory location that no longer contains a MenuItem (at which point the cortex becomes crash-happy)